demos/yolov5/README.md

```diff
@@ -0,0 +1,26 @@
+# YOLOv5 Example
+
+Simplest possible example of tracking. Based on <a href="https://github.com/fcakyon/yolov5-pip">YOLOv5-pip</a>.
+
+## Instructions 
+
+1. Install Norfair with `pip install norfair[video]`.
+2. Install YOLOv5 with `pip install yolov5`.
+3. Copy `yolov5demo.py` into your local and run `python yolov5demo.py <video file>`.
+4. Bonus: Use additional arguments `--detector_path`, `--img_size`, `--iou_thres`,`--conf_thres`, `--classes`, `--track_points` as you wish.
+
+## Explanation
+
+This example tracks objects using a single point per detection: the centroid of the bounding boxes around objects returned by YOLOv5.
```

Comment:
This is now only partially correct since you added the `--track_points` option ;)